[BUG] HTTPError 403: Forbidden

** Main info: **

`-s 25169 -e 13 -q 1080p --dlsubs frFR -t 'Kami no Tou' -x 3 --tsparts 1`

> === Crunchyroll Downloader NX 4.18.3 ===
> 
> [ERROR] HTTPError 403: Forbidden

** Additional Info: **

I got the same issue when trying to log in. Yesterday was everything fine but I tried today and its not working anymore

Log :

`[ERROR] HTTPError 403: Forbidden
[ERROR] Authentication failed!
[AUTH] Body: <!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en-US"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en-US"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en-US"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en-US"> <!--<![endif]-->
<head>
<title>Attention Required! | Cloudflare</title>
<meta name="captcha-bypass" id="captcha-bypass" />
<meta charset="UTF-8" />
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1" />
<meta name="robots" content="noindex, nofollow" />
<meta name="viewport" content="width=device-width,initial-scale=1" />
<link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/cf.errors.css" type="text/css" media="screen,projection" />
<!--[if lt IE 9]><link rel="stylesheet" id='cf_styles-ie-css' href="/cdn-cgi/styles/cf.errors.ie.css" type="text/css" media="screen,projection" /><![endif]-->
<style type="text/css">body{margin:0;padding:0}</style>


<!--[if gte IE 10]><!--><script type="text/javascript" src="/cdn-cgi/scripts/zepto.min.js"></script><!--<![endif]-->
<!--[if gte IE 10]><!--><script type="text/javascript" src="/cdn-cgi/scripts/cf.common.js"></script><!--<![endif]-->





<style type="text/css">
  #cf-wrapper #spinner {width:69px; margin:  auto;}
  #cf-wrapper #cf-please-wait{text-align:center}
  .attribution {margin-top: 32px;}
  .bubbles { background-color: #f58220; width:20px; height: 20px; margin:2px; border-radius:100%; display:inline-block; }
  #cf-wrapper #challenge-form { padding-top:25px; padding-bottom:25px; }
  #cf-hcaptcha-container { text-align:center;}
  #cf-hcaptcha-container iframe { display: inline-block;}
  @keyframes fader     { 0% {opacity: 0.2;} 50% {opacity: 1.0;} 100% {opacity: 0.2;} }
  #cf-wrapper #cf-bubbles { width:69px; }
  @-webkit-keyframes fader { 0% {opacity: 0.2;} 50% {opacity: 1.0;} 100% {opacity: 0.2;} }
  #cf-bubbles > .bubbles { animation: fader 1.6s infinite;}
  #cf-bubbles > .bubbles:nth-child(2) { animation-delay: .2s;}
  #cf-bubbles > .bubbles:nth-child(3) { animation-delay: .4s;}
</style>
</head>
<body>
  <div id="cf-wrapper">
    <div class="cf-alert cf-alert-error cf-cookie-error" id="cookie-alert" data-translate="enable_cookies">Please enable cookies.</div>
    <div id="cf-error-details" class="cf-error-details-wrapper">
      <div class="cf-wrapper cf-header cf-error-overview">
        <h1 data-translate="challenge_headline">One more step</h1>
        <h2 class="cf-subheadline"><span data-translate="complete_sec_check">Please complete the security check to access</span> www.crunchyroll.com</h2>
      </div><!-- /.header -->

      <div class="cf-section cf-highlight cf-captcha-container">
        <div class="cf-wrapper">
          <div class="cf-columns two">
            <div class="cf-column">

              <div class="cf-highlight-inverse cf-form-stacked">
                <form class="challenge-form" id="challenge-form" action="/xml/?req=RpcApiUser_Login&amp;__cf_chl_captcha_tk__=53f5c6719267b41141ec22d01a4492d00522d2f3-1593027493-0-AcDh5rHM0ZZUE5FhM3STGRGz3GH6tWLmetnI6zJsyHqai3JA1bALWKAAUTHPs7WdTItFPzUZm5Fw-GVcrtqwYkQ85R2gW8Odcxs6J9mdwEfCrrTkUcTW37GyT2rLS3fi9E0Q-gjdJRekdv5e-lKVQIQnufMo_BFXZZsb5GyuX-FNP9Z768xEkjmajEd6WsOgttikYJaT9UGMTKdJiZe51RDW5M4mW2UzWcp2F9zLkshDNWfcxF38510h2ENdC9dxTSJUcHCSmzZO39rnRgbFN71jUrFxs0kxZCm312SudVgQ9ASayyHuQiogYRdaQm_98p_-eeUZzmVurlrXvoVQLNiimDqsK13dZf--lUvWt-zsbK-x_rW4K6twr4j4a4ivcLSMfvsAKqbdpXnrwg_88d-LgTHbpJhVOqTCILm0Rkg6lb13PmI5qD2gCbq-Q-pHq3hmsnreBQ56FdUDj-oxL6rykXnZxKJ6cW8WIkt0EFU4wiwJ5U6MQBy3w6wgHPE3d2CO-2xvmwy-FHMeSseu2KqPOhZ6_H-zVe0q2JIQ-ApuMaAH5Do5qLZ87kkznUSKaynXjxWRB9Zg6iaSJtUqmTM" method="POST" enctype="application/x-www-form-urlencoded">

  <input type="hidden" name="r" value="a1fa39fd68a1a961c2f5ebd13546da738e9ebdc4-1593027493-0-AfKIxgKPy0RcmWh02tn862ioQfre1OkShkcwE6LoC41ei1UsvDRlchwAdBvYsur01wN8jd0X51XyIYwFshswFrSh6KRDVopladD7sRtLlwGAPz1d4NPk2y9dV/j6+dpZRylFUX9slysgB0/zwlV7R/Wn07To442FETCH9jETo8db9ChVru0g5Lgr6f/QAZt1qtrXWenf5hMxfxKM4NyRerAvnTZLHMyUQPPmc73LlWoV/FxqwuClw3Dnuli2sR13TolZu1bvGhlo7MFh/cJxLrV8eoNKUTSKybQJwr/xPWlSjBaqknyPIQccY4fBVA7XMdnnzMZ4WZJmjhoJLxwcTqTpXAfKWJLf9Np7XciIDJiDcIjBOVjl9OTD0eW0/rtHBqiZkQOOCw4K41pZxGax/89xDmoHkYWnmFuXUJRzUPtQadIElacgI2Wgx2KzAtTGfEZZ8k1eGAUq4CBbBIiUF9KPs+LpFhthByUCwOFCQ4Cq/AiA1H086+IZMi2lIxrtz8a76GKPt1LdTTWmvdzX1cYvlldQAW0TXs8S+1ecrutp3A7QCtoC21TS36KyvRXyc6Zq94Rtri6O7oOYMPiGMIa0SsHf4xDW2k4HAv2tXKsrVd7BKUfxZwpZambKxk7D4ZTMQA9NNwswo8iFJqSN49gWY8KrDm5VYDvKnPA7jsabHekDt/3m6GPkdU7oHBU2qK+dsTdOuH8h5fl4L9KUgZwMynP5/o/NgXd+sLcg/vrU+DbaNQeQjZ4p1UOI5rf6N7b0F+YH07cCwIdxhrUjbqHALLu7hC6qy9cyCZwKOgF2XlmByVWeqqrXQ1lkFC1XQOGk2YRKR3gx2TvD+HfZoT1pTayuu4Nn7KYCQMr7ZlInDisK/0geLtbggfR0hjT0ZDOB6GaSJBDhMFSHQRk2/IOnPhI16xJqARgbHgP0m2wIsdqSFWTNaBpZ4mPYsGTiKhGmakoybo7A8oUKYiwWBDlR6IvWtbJVsTDovWYg38fDnVWrSEOWA5qqUJQMRqjHe7kSG89A/f5mIj/JFqIm2m6OJtzJey22PQbYbNlIOIEOu7WQ9fcDGEemJIG/PQDpaYA+b5UnhHRNwO8MvR2nMWyGJwkN/ZhSgbc9uVsAGefgbT5DyjQshxYKuo5seSH9dNjmhltvvtpfwnkrlgzdRPEhv0LlQ7Dsvd4FrrYlFSL7MchKkINSvParNkwZ3/V+Sr83dxE6M+eNVp0y4dhhpbWvi8+8hZMen/TLNoevCMcE/E/7dvTbSDJweB1xhUsG2CB8wEe+EfS6Xy/sLYMS5EgMIssbQXOT87uzP4s4p3jMtmc0nb33lPDNHkUgxphz59y4hu2xSMRV9DF6JOcqVrhU/CIBpWKyNjDPj5sAUGrIzNuevjCCMzJoVgtz4hb7Uw==">
  <input type="hidden" name="cf_captcha_kind" value="h">
  <input type="hidden" name="vc" value="">

  <script type="text/javascript" src="/cdn-cgi/scripts/hcaptcha.challenge.js" data-type="normal"  data-ray="5a88e86939e0e744" async data-sitekey="33f96e6a-38cd-421b-bb68-7806e1764460"></script>

  <noscript id="cf-captcha-bookmark" class="cf-captcha-info">
  <h1 data-translate="turn_on_js" style="color:#bd2426;">Please turn JavaScript on and reload the page.</h1>
  </noscript>
  <div id="no-cookie-warning" data-translate="turn_on_cookies" style="display:none">
    <h1 data-translate="turn_on_cookies" style="color:#bd2426;">Please enable Cookies.</h1>
  </div>
  <script type="text/javascript">
  //<![CDATA[
    var a = function() {try{return !!window.addEventListener} catch(e) {return !1} },
      b = function(b, c) {a() ? document.addEventListener("DOMContentLoaded", b, c) : document.attachEvent("onreadystatechange", b)};
      b(function(){
        var cookiesEnabled=(navigator.cookieEnabled)? true : false;
        if(!cookiesEnabled){
          var q = document.getElementById('no-cookie-warning');q.style.display = 'block';
        }
      });
  //]]>
  </script>
  <div id="trk_captcha_js" style="background-image:url('/cdn-cgi/images/trace/captcha/nojs/h/transparent.gif?ray=5a88e86939e0e744')"></div>
</form>

              </div>
            </div>

            <div class="cf-column">
              <div class="cf-screenshot-container">

                <span class="cf-no-screenshot"></span>

              </div>
            </div>
          </div><!-- /.columns -->
        </div>
      </div><!-- /.captcha-container -->

      <div class="cf-section cf-wrapper">
        <div class="cf-columns two">
          <div class="cf-column">
            <h2 data-translate="why_captcha_headline">Why do I have to complete a CAPTCHA?</h2>

            <p data-translate="why_captcha_detail">Completing the CAPTCHA proves you are a human and gives you temporary access to the web property.</p>
          </div>

          <div class="cf-column">
            <h2 data-translate="resolve_captcha_headline">What can I do to prevent this in the future?</h2>


            <p data-translate="resolve_captcha_antivirus">If you are on a personal connection, like at home, you can run an anti-virus scan on your device to make sure it is not infected with malware.</p>

            <p data-translate="resolve_captcha_network">If you are at an office or shared network, you can ask the network administrator to run a scan across the network looking for misconfigured or infected devices.</p>


              <p data-translate="resolve_captcha_privacy_pass"> Another way to prevent getting this page in the future is to use Privacy Pass. You may need to download version 2.0 now from the <a href="https://addons.mozilla.org/en-US/firefox/addon/privacy-pass/">Firefox Add-ons Store</a>.</p>


          </div>
        </div>
      </div><!-- /.section -->


      <div class="cf-error-footer cf-wrapper">
  <p>
    <span class="cf-footer-item">Cloudflare Ray ID: <strong>5a88e86939e0e744</strong></span>
    <span class="cf-footer-separator">&bull;</span>
    <span class="cf-footer-item"><span>Your IP</span>: 149.28.232.4</span>
    <span class="cf-footer-separator">&bull;</span>
    <span class="cf-footer-item"><span>Performance &amp; security by</span> <a href="https://www.cloudflare.com/5xx-error-landing?utm_source=error_footer" id="brand_link" target="_blank">Cloudflare</a></span>

  </p>
</div><!-- /.error-footer -->


    </div><!-- /#cf-error-details -->
  </div><!-- /#cf-wrapper -->

  <script type="text/javascript">
  window._cf_translation = {};


</script>


</body>
</html>`

--- a/crunchy.js
+++ b/crunchy.js
@@ -1374,6 +1374,7 @@ async function getData(durl, params){
         }
         options.minVersion = 'TLSv1.3';
         options.maxVersion = 'TLSv1.3';
+        options.http2 = true;
     }
     // debug
     options.hooks = {

